### ISSUE TITLE
Support for React 19

### ISSUE THREAD

**Issue by csvan** [enzymejs/enzyme#2611]
I am well aware that Enzyme has yet to officially support React 17 and 18, but adding this issue to track support of React 19,

**Comment by csvan** [comment]
We have long since dropped Enzyme altogether

**Comment by gerhardwimmer80** [comment]
Given that Enzyme relies on internal React `fiber` structures that have shifted significantly since version 17, are you planning to implement a new `Adapter` for React 19 or is this repository effectively sunsetted?

**Comment by ljharb** [comment]
@gerhardwimmer80 yes, every react major gets its own adapter. this repo is not sunsetted, it's just fallen quite far behind.
